packages/Main/src/Parser/Potree2BinParser.js

```diff
@@ -95,6 +97,9 @@ export default {
             }
         });
 
+        geometry.userData.origin = options.out.origin;
+        geometry.userData.rotation = new THREE.Quaternion();
+
```

Comment:
Transformation properties in geometry  : This doesn't seem to be the right place.